test/test_cursor.py

```diff
@@ -245,7 +253,7 @@ def test_max_await_time_ms(self):
         listener.reset()
 
         # Tailable_await with max_time_ms
-        list(coll.find(cursor_type=CursorType.TAILABLE_AWAIT).max_time_ms(99))
```

Comment:
We need to keep at least one test that uses `list(cursor)` to ensure that it's supported.